### PR TITLE
[React] Implemented Admin Section

### LINKED ISSUE #58: [Frontend] Implement Admin section
**Is your feature request related to a problem? Please describe.**
The idea is to create a separate admin section in the app which will implement all functions from the user routes of the flask server.

**Describe the solution you'd like**
The PR will consist of implementation of the following features -
1. View all users
2.  View the details of each user
3. Create a new user
4. Update the role of an existing user
5. Delete an existing user.

**Additional context**
I'd like to work on this

### PR DESCRIPTION
**Pull request by amlannandy**
# Description

Implemented the admin section in the React App. The following functionalities are available -
1. View the list of users
2. Create a new user
3. Delete an existing user
4. Update the role of an existing user

Fixes #58 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

1. Go to Admin Section
2. Create/Delete Users to test functionality

**Test Configuration**:

- Windows 10
- Chrome

screen-capture

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: React-frontend/src/pages/admin/AddUser.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import Dropdown from '../../components/core/Dropdown';
+
+const AddUser = ({ history }) => {
+  const dispatch = useDispatch();
+  const initialFormData = {
+    name: '',
+    email: '',
+    password: '',
+    role: 'admin',
+  };
```
Comment (sachinsom93) [inline React-frontend/src/pages/admin/AddUser.js:19]: Hey hi @amlannandy , great efforts, but why did you use role "admin" as a defualt role for an user, I think the roles of the users could be extractor or management ?

**Review by sachinsom93 — CHANGES_REQUESTED** [review]

File: React-frontend/src/pages/admin/AddUser.js (hunk 1 above)
Comment (amlannandy) [reply React-frontend/src/pages/admin/AddUser.js:19]: Hey! No specific reason for that. Admin is the first option in the dropdown menu. So I just put it as default. And thanks for pointing out the roles error! I only used the admin and user roles for testing, but forgot to change them later. Fixed them now. 👍🏻

**Comment by shivanshuraj1333** [comment]
Hi @amlannandy will review this PR in the evening, thanks.

**Comment by amlannandy** [comment]
> Hi @amlannandy will review this PR in the evening, thanks.

Sure! :)

**Review by shivanshuraj1333 — APPROVED** [review]

**Pull request merged**